force-app/main/default/classes/CredentialTrackingLWCAPEX.cls

```diff
@@ -0,0 +1,35 @@
+public class CredentialTrackingLWCAPEX {
+   @AuraEnabled
+   public static List<User_Credential__c> getUserCredentials(String userId)
+   {
+       UserCredentialService ucs = new UserCredentialService();
+       List<User_Credential__c> totalUserCredentials = ucs.getActiveUserCredentials(userId);
+       return totalUserCredentials;  
+   } 
+
+   
+   @AuraEnabled
+   public static Boolean updateUserCredential(String id, string status)
+   {
+       System.debug('NDEBUG Is is' + id);
+       System.debug('NDEBUG In updateUserCredential');
+        UserCredentialService ucs = new UserCredentialService();
+        List<User_Credential__c> userCredentials = new List<User_Credential__c>{new User_Credential__c(id=id,status__c=status)};
+        List<Database.SaveResult> srList = ucs.updateUserCredentials(userCredentials);
+        
+       System.debug('NDEBUG In updateUserCredential -- 1');
+        Boolean isSuccess= true;
```

Comment:
should this be set after checking for errors?

Reply:
Updated Changes please review